main/SetupMenu.cpp

```diff
@@ -980,12 +980,44 @@ void SetupMenu::audio_menu_create_volume( MenuEntry *top ){
 	top->addEntry( amspvol );
 }
 
+void SetupMenu::audio_menu_create_mute( MenuEntry *top ){
+	SetupMenuSelect * asida = new SetupMenuSelect( PROGMEM"In Sink", RST_NONE, 0 , true, &audio_mute_sink );
+	asida->setHelp(PROGMEM"Select whether vario audio will be muted while in sink");
+	asida->addEntry( PROGMEM"Stay On");  // 0
+	asida->addEntry( PROGMEM"Mute");     // 1
+	top->addEntry( asida );
+
+	SetupMenuSelect * ameda = new SetupMenuSelect( PROGMEM"In Setup", RST_NONE, 0 , true, &audio_mute_menu );
+	ameda->setHelp(PROGMEM"Select whether vario audio will be muted while Setup Menu is open");
+	ameda->addEntry( PROGMEM"Stay On");  // 0
+	ameda->addEntry( PROGMEM"Mute");     // 1
+	top->addEntry( ameda );
+
+	SetupMenuSelect * ageda = new SetupMenuSelect( PROGMEM"Generally", RST_NONE, 0 , true, &audio_mute_gen );
+	ageda->setHelp(PROGMEM"Select audio on, or vario audio muted, or all audio muted including alarms");
+	ageda->addEntry( PROGMEM"Audio On");      // 0
+	ageda->addEntry( PROGMEM"Alarms On");     // 1
+	ageda->addEntry( PROGMEM"Audio Off");     // 2
+	top->addEntry( ageda );
+
+	SetupMenuSelect * amps = new SetupMenuSelect( PROGMEM"Amplifier", RST_NONE, 0 , true, &amplifier_shutdown );
```

Comment:
How about the old mode of shutdown immediate? 
Note there is old hardware in field with some sizzling noise when this doesn't take place immediate. 
Propose a third option:
0 Stay On
1 Shutdown        -> immediate shutdown, old behavior
2 Idle Shutdown -> new: shutdown during long silence